src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -91,7 +98,7 @@ public void updateFilteredListToShowAll() {
 
     @Override
     public void updateFilteredTaskList(Set<String> keywords){
```

Comment:
No, we did not agree on this. @Skaty was under the assumption that you wouldn't mix the two together. Please do not do that in the future, as discussed earlier.